src/metalkit.cs

```diff
@@ -143,17 +154,52 @@ interface MTKTextureLoaderKeys {
 
 		[Field ("MTKTextureLoaderOptionTextureCPUCacheMode")]
 		NSString TextureCpuCacheModeKey { get; }
+
+		[iOS (10,0)][Mac (10,12, onlyOn64 : true)]
+		[Field ("MTKTextureLoaderOptionTextureStorageMode")]
+		NSString TextureStorageModeKey { get; }
+
+		[iOS (10,0)][Mac (10,12, onlyOn64 : true)]
+		[Field ("MTKTextureLoaderOptionCubeLayout")]
+		NSString CubeLayoutKey { get; }
+
+		[iOS (10,0)][Mac (10,12, onlyOn64 : true)]
+		[Field ("MTKTextureLoaderOptionOrigin")]
+		NSString OptionOriginKey { get; }
+	}
+
+	[iOS (10,0)][Mac (10,12, onlyOn64 : true)]
+	enum MTKTextureLoaderCubeLayout {
+		[Field ("MTKTextureLoaderCubeLayoutVertical")]
+		LayoutVertical,
```

Comment:
Layout is redundant here since the name of the enum is `MTKTextureLoaderCubeLayout`.